src/styles/index.scss

```diff
@@ -1,3 +1,42 @@
+@import './units/variables';
+@import './blocks/button-buy';
+@import './blocks/stars';
+@import './blocks/card-positions';
+@import './blocks/price';
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $cart-border-color;
+  border-radius: 5px;
+  background-color: $cart-bg-color;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0;
+    color: $title-color;
+  }
+
+  &__code {
+    color: $gray-color;
+    font-size: $font-S;
+    margin: 0;
+    line-height: 14px;
+  }
+}
```

Comment:
While the BEM methodology is used correctly, it seems that the styles are not separated into different files according to BEM blocks. According to the Code Style Rules, each BEM block should be in a separate file. Consider dividing your styles into separate files.